package.json

```diff
@@ -40,28 +40,25 @@
     },
     "dependencies": {
         "@azure/functions": "^1.2.3",
-        "@types/lodash": "^4.14.119",
-        "@types/node": "10.14.2",
-        "@types/uuid": "~3.4.4",
-        "@types/validator": "^9.4.3",

```

Comment:
No references found in *.d.ts files - moved to a dev dependency
<img width="591" alt="Screen Shot 2022-04-22 at 2 44 56 PM" src="https://user-images.githubusercontent.com/11282622/164802620-6bab4371-81ed-4cb7-addf-4bbfd38e9052.png">
.